r/work.R

```diff
@@ -41,7 +41,33 @@ run_post <- function(req, post_fun, data) {
     debug_step <- Sys.getenv("DEBUG_STEP", unset = "")
 
     handle_debug(req, debug_step)
-    post_fun(req, data)
+
```

Comment:
perhaps we could add some message to inform when experiment loading begins?

Reply:
that's already there, on line 228 and on the load_data function